src/mappings/entities/identities/mapClaim.ts

```diff
@@ -71,6 +71,23 @@ const getId = (
   return idAttributes.join('/');
 };
 
+const processClaim = async (claimScope: any, block: SubstrateBlock): Promise<Scope> => {
+  const scope = JSON.parse(claimScope);
+
+  if (scope.type === ClaimScopeTypeEnum.Ticker || scope.type === ClaimScopeTypeEnum.Asset) {
+    scope.type = ClaimScopeTypeEnum.Asset;
+    const { assetId, ticker } = await getAssetIdWithTicker(scope.value, block);
+
+    if (block.specVersion < 7000000) {
+      scope.value = ticker;
+    }
```

Comment:
always setting this value is no harm. Before specVersion 7000000, ticker value will always be there and for cases after 7000000 where ticker is not linked to asset, value will remain null. If you want to avoid setting `value: undefined` maybe the check can be modified as 
```suggestion
    if (ticker) {
      scope.value = ticker;
    }
```
and add comment that value can be null  if ticker is not liked with asset

Reply:
With ticker linked/unlinked events I didn't want these to come out of sync.

I'll continue setting it here for now since I think the entity refactor story normalizes the DB to make it not a concern.